src/content/docs/integrations/amazon-integrations/aws-integrations-list/aws-health-monitoring-integration.mdx

```diff
@@ -40,13 +40,141 @@ You can change the polling frequency and filter data using [configuration option
 
 ## Explore integration data [#find-use]
 
-This integration does not fetch metrics, only [inventory information](#metrics). To use this integration's data:
+To use this integration's data:
 
-1. Go to [**one.newrelic.com**](http://one.newrelic.com) **> Infrastructure > AWS**.
+1. Go to [**one.newrelic.com**](https://one.newrelic.com) **> Infrastructure > AWS**.
 2. Select any of the available AWS Health integration links.
 3. To view a complete list of open issues, select the [**Inventory** page](/docs/infrastructure/new-relic-infrastructure/infrastructure-ui-pages/infrastructure-inventory-page-search-your-entire-infrastructure).
 4. To view a timeline when an AWS Health inventory event is created, modified, or deleted, use the [**Events** page](/docs/infrastructure/new-relic-infrastructure/infrastructure-ui-pages/infrastructure-events-page-live-feed-every-config-change).
 
+## Events monitoring and alerts [#event-monitoring]
+
+AWS Health events are ingested in New Relic One as **AwsHealthNotification** events. You can query, create widgets, and define NRQL alert policies based on these events.
+
+The following table shows the main attributes available for **AwsHealthNotification** events:
+<table>
```

Comment:
Unfortunately, the markdown system we're using get's a little confused when there is text on the line directly before an opening HTML tag. This can be resolved by adding a new line:
```suggestion
The following table shows the main attributes available for **AwsHealthNotification** events:

<table>
```